types/comments-and-replies.ts

```diff
@@ -283,7 +283,7 @@ export interface _Comment {
    * 
    * @optional
    */
-  hateSpeech: ContentHateSpeechResult
+  hateSpeech?: ContentHateSpeechResult
```

Comment:
this will never be an optional

Reply:
wait, was this set here like this so that the example comment you created didnt throw type errors during development? if so then ignore this comment, just revert it b4 finalizing the pr

Reply:
im retarded sorry
it wont be optional
we can remove @optional